实验2部分/shiyan2_zonghe.py

```
@@ -0,0 +1,357 @@
+                np.savez(output_path, 
+                         mfcc=mfcc_features, 
+                         sample_rate=sample_rate,
+                         n_mfcc=N_MFCC,
+                         n_mels=N_MELS)
+                
+                print(f"  处理完成。采样率: {sample_rate} Hz")
+                print(f"  输出MFCC特征 E(i, n) 形状: {mfcc_features.shape}")
+                print(f"  [成功] 已保存至: {output_path}")
+                success_count += 1
+                
+            except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)